README.md

```diff
@@ -56,3 +56,60 @@ yarn run cypress
 ```
 yarn test:headless
 ```
+
+## UI Design
```

Comment:
Why don't you add something like: This is the structure of the React components in the app.